aws-cli/bash-linux/s3/bucket-lifecycle-operations/README.md

```diff
@@ -0,0 +1,43 @@
+.. Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+   This work is licensed under a Creative Commons Attribution-NonCommercial-ShareAlike 4.0    International License (the "License"). You may not use this file except in compliance with the    License. A copy of the License is located at http://creativecommons.org/licenses/by-nc-sa/4.0/.
+
+   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+
+# Amazon S3 Bucket Lifecycle Operations
+
+## Purpose
+
+This example demonstrates how to interact with some of the basic Amazon S3 operations. The script includes functions that perform the following tasks:
+
+ * Creating a bucket and verifying that it exists
+ * Copying a file from the local computer to a bucket
+ * Copying a file from one bucket location to a different bucket location
+ * Listing the contents of a bucket
+ * Deleting a file from a bucket
+ * Deleting a bucket
+
+## Prerequisites
+
+ * An Amazon Web Services (AWS) account.
+ * A shared credentials file with a default profile. The profile that you use must have permissions that allow the AWS operations performed by the script. For more information about how to set up a shared credentials file, see [Configuration and Credential File Settings](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html) in the _AWS CLI User Guide_.
```

Comment:
I'm not sure we need to specify AWS account and credentials as prerequisites for every example (because they're needed for pretty much every example), but it's your call. If you think your users want this, go ahead and keep it.

Reply:
Yeah, if they drop into here straight from google, I think it's good to call it out and include links to where they can get the info needed to make it happen.